src/containers/track-page/TrackPageProvider.tsx

```diff
@@ -409,10 +412,12 @@ class TrackPageProvider extends Component<
     })
     const canonicalUrl = user && track ? fullTrackPage(track.permalink) : ''
 
-    // If the track has a remix parent and it's not deleted.
+    // If the track has a remix parent and it's not deleted and the original's owner is not deactivated.
     const hasValidRemixParent =
       !!getRemixParentTrackId(track) &&
-      (!remixParentTrack || remixParentTrack.is_delete === false)
+      !!remixParentTrack &&
```

Comment:
Did the logic change a bit here (outside of deactivated)?. In the previous code if there is !remixParentTrack is falsy it's still possible for hasValidRemixParent to be true. In your change that's no longer the case

Reply:
Yeah the logic changed here, good catch, but I think it's more correct per the comment above? Not super familiar with this line.